src/services/blockchain/__tests__/__snapshots__/eth-bc-service.test.ts.snap

```diff
@@ -1,10 +1,10 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`ETH service connected to ganache should send CID to local ganache server 1`] = `
+exports[`ETH service connected to ganache v0 should send CID to local ganache server 1`] = `
 Transaction {
-  "blockNumber": 1,
+  "blockNumber": 3,
```

Comment:
Actually the ganache was originally set up to automine blocks every 2 seconds (found here: https://github.com/ceramicnetwork/ceramic-anchor-service/blob/1189f96529da41b4792a32a2c71f8bb0997f330b/src/services/blockchain/__tests__/eth-bc-service.test.ts#L62). 

So the block number went up because the set up takes longer with deploying the smart contract.